examples/siro_sandbox/controllers/fetch_baselines_controller.py

```diff
@@ -411,10 +417,37 @@ def act(self, obs, env):
         human_pos = env._sim.agents_mgr[
             1 - self._agent_idx
         ].articulated_agent.base_transformation.translation
+        human_rot = env._sim.agents_mgr[
+            1 - self._agent_idx
+        ].articulated_agent.base_rot
         robot_pos = env._sim.agents_mgr[
             self._agent_idx
         ].articulated_agent.base_transformation.translation
 
+        # Check if the human walks or rotates
+        is_human_walk = (
+            np.linalg.norm(
+                np.array((self._prev_human_pos - human_pos))[[0, 2]]
+            )
+            > 0.01
```

Comment:
Can you expose this as a global at top of file? I will need to tune it for VR, where the headset is always moving slightly.

Reply:
Thank you, Eric! I added `DISTANCE_ROTATION_TRESHOLD_HUMAN_MOVING` for this.